File: problems/first-greater-number/first-greater-number.py
def first_greater(l, n):
  return next((i for i in l if i > n), None)

print first_greater([2, 10,5,6,80], 6)
print first_greater([2, 10,5,6,80], 20)
print first_greater([2, 10,5,6,80], 100)

File: problems/first-greater-number/first-greater-number.js
const firstGreater = (arr, n) => arr.find(num => num > n);

console.log(firstGreater([2, 10,5,6,80], 6));   // 10
console.log(firstGreater([2, 10,5,6,80], 20));  // 80
console.log(firstGreater([2, 10,5,6,80], 100)); // undefined